lib/v2/untag/category.js

```diff
@@ -0,0 +1,65 @@
+const got = require('@/utils/got');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    const category = ctx.params.category ?? 'method';
+    const limit = ctx.query.limit ? parseInt(ctx.query.limit) : 20;
+
+    const rootUrl = 'https://utgd.net';
+    const apiUrl = `${rootUrl}/api/v2/pages`;
+    const currentUrl = `${rootUrl}/category/s/${category}`;
+    const slugUrl = `${rootUrl}/api/v2/category/slug/${category}`;
```

Comment:
```suggestion
    const apiUrl = `${rootUrl}/api/v2/pages/`;
    const currentUrl = `${rootUrl}/category/s/${category}`;
    const slugUrl = `${rootUrl}/api/v2/category/slug/${category}/`;
```
https://github.com/DIYgod/RSSHub/actions/runs/5105246772/jobs/9176711212#step:12:17
https://github.com/DIYgod/RSSHub/actions/runs/5105246772/jobs/9176711212#step:12:19

Reply:
👌